contracts/rewards/src/contract.rs

```diff
@@ -25,7 +25,7 @@ pub fn migrate(
     _env: Env,
     _msg: Empty,
 ) -> Result<Response, axelar_wasm_std::error::ContractError> {
-    v0_4_0::migrate(deps.storage)?;
+    v1_0_0::migrate(deps.storage)?;
```

Comment:
should we move the `set_contract_version` call into the migration after all?

Reply:
I'd rather leave it here, since some migrations are just code updates and have no migration logic, so the only place to set the version would be here. I'd rather it just always be set here, to keep things simple.